src/controller/java/src/matter/onboardingpayload/OptionalQRCodeInfo.kt

```diff
@@ -37,4 +37,40 @@ class OptionalQRCodeInfoExtension : OptionalQRCodeInfo() {
   var int64: Long = 0
   var uint32: Long = 0
   var uint64: Long = 0
+
+  override fun equals(other: Any?): Boolean {
+    if (this === other) return true
+    if (other !is OptionalQRCodeInfoExtension) return false
+
+    return tag == other.tag &&
+      type == other.type &&
```

Comment:
Those properties are not defined in OptionalQRCodeInfoExtension, but OptionalQRCodeInfo, those should be checked in the parent class, otherwise, this check will fail if parent class add another property later

Reply:
Good point. I'll get that fixed up